src/main/java/uk/ac/ebi/eva/pipeline/parameters/validation/job/GenotypedVcfJobParametersValidator.java

```diff
@@ -52,6 +53,8 @@ private CompositeJobParametersValidator compositeJobParametersValidator(JobParam
 
         Boolean skipAnnotation = Boolean.valueOf(jobParameters.getString(JobParametersNames.ANNOTATION_SKIP));
         if (!skipAnnotation) {
+            jobParametersValidators.add(new InputStudyIdValidator());
```

Comment:
Why has this been taken out of the step, is it because it depends on the job instead of the step? Could a comment be added to clarify it?